pantheon-bundle/frontend/src/app/versions.tsx

```diff
@@ -532,12 +530,13 @@ class Versions extends Component<IProps, IState> {
                 }).then(response => {
                     if (response.status === 201 || response.status === 200) {
                         console.log(buttonText + " works: " + response.status)
+                        console.log(" response is : " + response.url)
+                        this.props.getUrl(response.url)
```

Comment:
Suggestion: getUrl is a bit confusing for a method that sets or raises event for URL. Consider renaming.